### PR TITLE
fix(i18n): add missing translations

### PR DESCRIPTION
**Pull request by brofik**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-11058 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

**Review by anooparveti — APPROVED** [review]

File: packages/manager/tools/sao-ovh-manager-module/template/src/onboarding/translations/Messages_fr_FR.json
Comment (antleblanc) [inline packages/manager/tools/sao-ovh-manager-module/template/src/onboarding/translations/Messages_fr_FR.json:1]: **question**: Why removing default translation files from the actual generator?
same question for `packages/manager/tools/sao-ovh-manager-module/template/src/translations/Messages_fr_FR.json`.

**Review by antleblanc — CHANGES_REQUESTED** [review]
Could you please resolve existing conflicts?

I've left one question regarding the generator.

File: packages/manager/tools/sao-ovh-manager-module/template/src/onboarding/translations/Messages_fr_FR.json
Comment (brofik) [reply packages/manager/tools/sao-ovh-manager-module/template/src/onboarding/translations/Messages_fr_FR.json:1]: Because these files are empty.

Comment (antleblanc) [reply packages/manager/tools/sao-ovh-manager-module/template/src/onboarding/translations/Messages_fr_FR.json:1]: Theses files must be kept. Could you please revert changes? Thank you

**Review by antleblanc — CHANGES_REQUESTED** [review]
Some translation files must be kept (generator).

File: packages/manager/tools/sao-ovh-manager-module/template/src/onboarding/translations/Messages_fr_FR.json
Comment (brofik) [reply packages/manager/tools/sao-ovh-manager-module/template/src/onboarding/translations/Messages_fr_FR.json:1]: I've done the revert on the commit that remove fr translation files.

**Review by antleblanc — DISMISSED** [review]
Some translation files must be kept (generator).

**Review by anooparveti — DISMISSED** [review]

**Comment by brofik** [comment]
> Some translation files must be kept (generator).

which ones ?

**Review by anooparveti — CHANGES_REQUESTED** [review]
I think this commit cd6ee06bc3ef729f021829c4d6586146ba5ef5c8 can be removed. We dont know the consequences of removal of keys. Maybe "empty string" is expected for this keys.
Can you confirm this dont have any consequence?

**Comment by brofik** [comment]
> I think this commit [cd6ee06](https://github.com/ovh/manager/commit/cd6ee06bc3ef729f021829c4d6586146ba5ef5c8) can be removed. We dont know the consequences of removal of keys. Maybe "empty string" is expected for this keys. Can you confirm this dont have any consequence?

I don't see these labels used anywhere. So I don't see the point of keeping them.

**Review by anooparveti — APPROVED** [review]

**Review by aboungnaseng-ovhcloud — APPROVED** [review]
👍

**Pull request merged**